code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -278,7 +278,8 @@
 			else if(!key)
 				msg += "<span class='deadsay'>[t_He] [t_is] totally catatonic. The stresses of life in deep-space must have been too much for [t_him]. Any recovery is unlikely.</span>\n"
 			else if(!client)
-				msg += "[t_He] [t_has] a vacant, braindead stare...\n"
+				msg += "[t_He] [t_has] suddenly fallen asleep, suffering from Space Sleep Disorder.\n"
+				msg += "Find somewhere safe for [t_him], [t_He] may wake up soon.\n"

```

Comment:
Sleeping is bad, I am pretty sure you can get sharded while sleeping(even if you don't have a client), and it will be really shit that happening.